### ISSUE TITLE
Loops add go to Prior/next error

### ISSUE THREAD

**Issue by tjohnson1010** [Azure/LogicAppsUX#6617]
### Describe the issue

When debugging a Run with any kind of loop, the loops have the option of a next or prior iteration.  Can you add a go to next or prior error?  Maybe use "<<" and ">>" for them.  The idea if an iteration has an error, you can click the ">>" and it will go to the next iteration with an error.  Right now, if you have 100 iterations, and one had failed, you will need to click through each one and wait to see if it was the one reported as an error.

### Are you using new designer or old designer

New Designer (With action panels)

### Screenshots or Videos

![Image](https://github.com/user-attachments/assets/95c79ec2-15e9-4ee7-9c52-3201760bdaa0)

### Browser

Microsoft Edge for Business
Version 133.0.3065.69 (Official build) (64-bit)

**Comment by ccastrotrejo** [comment]
Hi @tjohnson1010 thanks for letting us know about this issue. We do have a pager that goes directly to the failed iteration. I tried to reproduce your workflow using similar actions from your screenshot and was able to see the failed pager. We render the failed pager based on the data we get from an API call, so it might be that the API call is failing.

- Is there any chance you can share with us the HAR file wile you load the run? Here is a small [guide](https://agreeable-beach-04ed7ce1e.5.azurestaticapps.net/tsg#collecting-network-logs-in-a-har-file) on how to get the HAR file
- Is there any chance you can share your workflow.json?

You can share both items above to my emaul ccastrotrejo@microsoft.com

https://github.com/user-attachments/assets/c75806aa-119e-47f8-a890-f9edd876f488

**Comment by tjohnson1010** [comment]
I sent you the Har and JSON for the Logic App.
FYI:  The instructions to get ethe HAR are out of date for Edge.
Thank you.

**Comment by tjohnson1010** [comment]
I sent the requested information and updated this ticket saying I did.  what else is there to do?

**Comment by ccastrotrejo** [comment]
Hi @tjohnson1010 sorry for not replying earlier, I just had time to take a look at it. I looked at your HAR file, I will reply you by email
